polaris.shopify.com/src/components/SideMenu/SideMenu.module.scss

```diff
@@ -0,0 +1,99 @@
+@import "../../styles/variables.scss";
+
+.SideMenu {
+  display: flex;
+  visibility: hidden;
+  align-items: flex-start;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: calc(100vw - 4rem);
+  max-width: 22.5rem;
+  min-width: 15rem;
+  background: var(--surface);
+  font-size: var(--font-size-100);
+  font-weight: 500;
+  z-index: 2;
+  transform: translateX(-22.5rem);
+
+  &.show {
+    visibility: visible;
+    transform: translateX(0);
+    transition: transform var(--duration-200) var(--ease-in-out);
+
+    .CloseButton {
+      position: absolute;
+      display: flex;
+      top: calc(
+        var(--header-height) / 2 - 12px
+      ); // 12px is half the height of the Close Icon
+      right: calc(-24px - 1rem); // 24px is the width of the Close Icon
```

Comment:
Rather than comments, could this be communicated through code? By extracting the icon size into a CSS variable at the top? `--mobile-nav-icon-size` and then using `calc` to factor it in.